docs/Performance and Tuning/Hardware.rst

```diff
@@ -588,6 +589,7 @@ follows:
 -  Intel 710
 -  Intel 730
 -  Intel DC S3500/S3510/S3610/S3700/S3710
+-  Kingston DC450R/DC500R/DC500M
```

Comment:
Could you please point at some source? https://www.kingston.com/en/ssd/dc450-data-center-solid-state-drive says at least DC450R doesn't have PLP.

Reply:
Oups, I double checked the DC450R and deleted it but I pulled the wrong commit. My mistake.